components/ShoppingItem/ShoppingItem.js

```diff
@@ -1,37 +1,53 @@
-import styled from "styled-components";
+import styled, { ThemeConsumer } from "styled-components";
```

Comment:
You do not use the `ThemeConsumer`, so please delete it.

```suggestion
import styled from "styled-components";
```

Reply:
Done.